core/src/test/java/org/elasticsearch/search/aggregations/bucket/significant/InternalSignificantTermsTestCase.java

```diff
@@ -91,14 +143,22 @@ protected void assertBucket(MultiBucketsAggregation.Bucket expected, MultiBucket
 
         assertEquals(expectedSigTerm.getSignificanceScore(), actualSigTerm.getSignificanceScore(), 0.0);
         assertEquals(expectedSigTerm.getSubsetDf(), actualSigTerm.getSubsetDf());
+        assertEquals(actualSigTerm.getDocCount(), actualSigTerm.getSubsetDf());
```

Comment:
By definition, getDocCount() returns the subset df value. The test used to test that getSubsetDf() return the same value for both the internal and parsed aggregations, but it didn't test that getDocCount() returns the susbset df as the internal implementation does. So I added this assertion.